functions.py

```diff
@@ -29,3 +32,19 @@ def getUserInput(prompt, isSecret=False):
     except KeyboardInterrupt:
         quit()
 
+def clipboard(text,prnt, clear):
+    pyperclip.copy(text)
+    print("Copied to clipboard")
+    if prnt:
+        print(text)
+    if clear:
+        myThread = Thread(target=timer, args=(30,))
+        myThread.start()
+
+def timer(seconds):
+    sleep(seconds)
+    clearclip()
+
+def clearclip():
+    pyperclip.copy("")
```

Comment:
Is there any way to check if the clipboard still contains the password before setting it to ""?

I could imagine logging in and copying something quickly, only to have my clipboard overwritten 30 seconds after login before I had a chance to paste.